### PR TITLE
Support loading conversation in ConversationActivity and Conversation…

### PR DESCRIPTION
**Pull request by howawong**
…Fragment by supplying conversation ID in intent

1. ConversationFragment and ConversationActivity accept conversation ID
2. call chat API and refresh UI once conversation is loaded via getConversation

Connects #183

### CODE REVIEW

File: chat_ui/src/main/java/io/skygear/plugins/chat/ui/ConversationFragment.kt [hunk 1]
```diff
@@ -233,14 +261,24 @@ open class ConversationFragment() :
...
+    fun refresh() {
         if (conversationView()?.itemCount() == 0) {
             this.conversation?.let {
+                updateTitle()
                 this.fetchUnsentMessages()
                 this.fetchMessages()
             }
         }
+        this.updateTitle()
+        this.fetchParticipants()
```
[the remark this answers is not in the document]
Comment (howawong) [reply chat_ui/src/main/java/io/skygear/plugins/chat/ui/ConversationFragment.kt]: It is fine, `updateTitle` will be called by the end of `fetchParticipants`.

File: chat_ui/src/main/java/io/skygear/plugins/chat/ui/ConversationFragment.kt [hunk 2]
```diff
@@ -195,7 +206,27 @@ open class ConversationFragment() :
         view.setOnMessageLongClickListener(this)
         view.setVoiceMessageOnClickListener(this)
         view.setLoadMoreListener(this)
-        view.setConversation(conversation)
+        if (conversation != null) {
+            view.setConversation(conversation)
+        } else {
+            conversationId?.let { conversationId ->
```
[the remark this answers is not in the document]
Comment (howawong) [reply chat_ui/src/main/java/io/skygear/plugins/chat/ui/ConversationFragment.kt]: added

File: chat_ui/src/main/java/io/skygear/plugins/chat/ui/ConversationView.kt [hunk 3]
```diff
@@ -340,6 +340,10 @@ open class ConversationView : RelativeLayout {
         this.progressBar?.visibility = View.GONE
     }
 
+    open fun showProgress() {
+        this.progressBar?.visibility = View.GONE
```
Comment (carmenlau) [inline chat_ui/src/main/java/io/skygear/plugins/chat/ui/ConversationView.kt]: showProgress, but `visibility = View.GONE`?

**Comment by carmenlau** [comment]
@howawong Please fix this so that we can merge and update the demo. thanks!!!

**Pull request merged**